pom.xml

```diff
@@ -619,7 +619,7 @@
         <junit.platform.version>1.9.3</junit.platform.version>
         <junit.vintage.version>5.9.3</junit.vintage.version>
         <concurrent.version>1.3.4</concurrent.version>
-        <xbean-spring.version>4.23</xbean-spring.version>
+        <xbean-spring.version>4.21</xbean-spring.version>
```

Comment:
Anything wrong with that?

Reply:
CI fail at Spring module
https://ci-builds.apache.org/blue/organizations/jenkins/james%2FApacheJames/detail/PR-1655/2/tests/

```
Error creating bean with name 'serverConnector' defined in class path resource [META-INF/org/apache/james/spring-server.xml]: Invocation of init method failed; nested exception is java.io.IOException: Cannot bind to URL [rmi://127.0.0.1:9999/jmxrmi]: javax.naming.CommunicationException [Root exception is java.rmi.ServerException: RemoteException occurred in server thread; nested exception is: 
 java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
 java.io.InvalidClassException: filter status: REJECTED]
Stacktrace
org.springframework.beans.factory.BeanCreationException: 
Error creating bean with name 'serverConnector' defined in class path resource [META-INF/org/apache/james/spring-server.xml]: Invocation of init method failed; nested exception is java.io.IOException: Cannot bind to URL [rmi://127.0.0.1:9999/jmxrmi]: javax.naming.CommunicationException [Root exception is java.rmi.ServerException: RemoteException occurred in server thread; nested exception is: 
	java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
	java.io.InvalidClassException: filter status: REJECTED]
	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:26)
Caused by: java.io.IOException: 
Cannot bind to URL [rmi://127.0.0.1:9999/jmxrmi]: javax.naming.CommunicationException [Root exception is java.rmi.ServerException: RemoteException occurred in server thread; nested exception is: 
	java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
	java.io.InvalidClassException: filter status: REJECTED]
	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:26)
Caused by: javax.naming.CommunicationException
	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:26)
Caused by: java.rmi.ServerException: 
RemoteException occurred in server thread; nested exception is: 
	java.rmi.UnmarshalException: error unmarshalling arguments; nested exception is: 
	java.io.InvalidClassException: filter status: REJECTED
	at org.apache.james.app.spring.JamesAppSpringMainTest.testServer(JamesAppSpringMainTest.java:26)
Caused by: java.rmi.UnmarshalException: 
error unmarshalling arguments; nested exception is: 
	java.io.InvalidClassException: filter status: REJECTED
Caused by: java.io.InvalidClassException: filter status: REJECTED
```